### PR TITLE
docs: add forge script

### PR DESCRIPTION
**Pull request by shawnharmsen**
closes: https://github.com/foundry-rs/book/issues/332

1. Should `forge script` be under `References/forge Commands/Deploy Commands` or `Utility Commands` like `forge run` ?
2. `forge run` should be changed to `forge debug` I think, maybe address in separate issue

I just noticed that `forge script` is using `multi_wallets.rs`
https://github.com/foundry-rs/foundry/blob/b192af49c6e1f3874bc6775cfd969040fddce740/cli/src/opts/multi_wallet.rs#L86

for its wallet options rather than `wallet.rs`
https://github.com/foundry-rs/foundry/blob/b192af49c6e1f3874bc6775cfd969040fddce740/cli/src/opts/wallet.rs#L52

~~I will update docs shortly~~

## changes
- added `src/reference/common/multi-wallet-options.md` with plural and different short flag for `--froms`

### CODE REVIEW

File: src/reference/forge/forge-script.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+Run a smart contract as a script, building transactions that can be sent onchain.
+Adds the ability for forge to use a solidity file to create a contract on mainnet and arbitrary deploy scripts by adding a cheatcode.
+
+### OPTIONS
+
+`--broadcast`  
+&nbsp;&nbsp;&nbsp;&nbsp;Broadcasts the transactions.
+
+`--debug`  
+&nbsp;&nbsp;&nbsp;&nbsp;Takes precedence over broadcast.
```
Comment (onbjerg) [inline src/reference/forge/forge-script.md:22]: What does it do? If this is the description of the flag in Forge we should fix that upstream as well as this is pretty non-descriptive

File: src/reference/forge/forge-script.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+`--sig` *signature*  
+&nbsp;&nbsp;&nbsp;&nbsp;The signature of the function you want to call in the contract, or raw calldata.  
+&nbsp;&nbsp;&nbsp;&nbsp;Default: `run()`  
+
+`--slow`  
+&nbsp;&nbsp;&nbsp;&nbsp;Makes sure a transaction is sent, only after its previous one has been confirmed and succeeded.
+
+`--target-contract` *contract_name*  
+&nbsp;&nbsp;&nbsp;&nbsp;The name of the contract you want to run
```
Comment (onbjerg) [inline src/reference/forge/forge-script.md:38]:
```suggestion
&nbsp;&nbsp;&nbsp;&nbsp;The name of the contract you want to run.
```

**Review by onbjerg — CHANGES_REQUESTED** [review]
Minor nits, however I think we should include the `forge debug` change in this PR as well. I also think `forge script` should probably be under "Utility Commands" for now, since it sort of replaces `forge run` (which is now just a way to run the debugger)

File: src/reference/forge/forge-script.md (hunk 1 above)
Comment (shawnharmsen) [reply src/reference/forge/forge-script.md:23]: It opens it in the debugger.

same results for both commands below

```bash
forge script ./test/Broadcast.t.sol --tc BroadcastTest --sig "deploy()" -vvv --fork-url $GOERLI_RPC_URL --debug
forge debug ./test/Broadcast.t.sol --tc BroadcastTest --sig "deploy()"
```

**Comment by shawnharmsen** [comment]
Think I got it all

1. Change `forge run` to `forge debug`
2. All `forge script` docs
3. `forge script --debug` help text in foundry pr: https://github.com/foundry-rs/foundry/pull/1782

**Review by onbjerg — APPROVED** [review]

**Pull request merged**